src/main/homework4/DZ1/CreditCard.java

```diff
@@ -0,0 +1,41 @@
+package main.homework4.DZ1;
+//1. Создать класс CreditCard c полями номер счета, текущая сумма на счету.
+//Добавьте метод, который позволяет начислять сумму на кредитную
+//карточку.
+//Добавьте метод, который позволяет снимать с карточки некоторую
+//сумму.
+//Добавьте метод, который выводит текущую информацию о карточке (номер счета, текущая сумма на счету).
+
+import java.util.Scanner;
+
+public class CreditCard {
+    private int account;
+    private int currentAmount;
+
+    public int getCurrentAmount()
+    {
+        return currentAmount;
+    }
+
+    CreditCard(int account, int currentAmount)
+    {
+        this.account = account;
+        this.currentAmount = currentAmount;
+    }
+
+    public void debit(int currentAmount)
+    {
+        this.currentAmount +=currentAmount;
+    }
+
+    public void credit(int currentAmount)
```

Comment:
1. What if we decide to get amount, which is bigger, than we have?
2. This method should return something.

Reply:
@marina-kostenko 
This method should return something.